eq/server/config/resources.cpp

```diff
@@ -312,7 +316,19 @@ bool Resources::discover( ServerPtr server, Config* config,
             name << "display";
         }
         else
+        {
             name << "GPU" << ++gpuCounter;
+#ifdef EQUALIZER_USE_HWSD_gpu_glx
+            if( info.flags & hwsd::GPUInfo::FLAG_VIRTUALGL &&
+                info.device != LB_UNDEFINED_UINT32 )
+            {
+                std::ostringstream displayString;
+                displayString << info.nodeName << ":" << info.port << "."
+                              << info.device;
```

Comment:
Shouldn't. VGL_EXCLUDE is only needed for local GPUs on the application node (which is were vglrun has been used) so there's non need to specify the host.